src/einsteinpy/symbolic/tensor.py

```diff
@@ -125,7 +127,10 @@ def __str__(self):
         Returns a String with a readable representation of the object of class Tensor
 
         """
-        representation = "Tensor" + "\n"
+        representation = "Tensor"
+        if self.name:
```

Comment:
```suggestion
        if self.name is not None:
```

This is a bit more clear :)